app/controllers/change_controller.rb

```diff
@@ -67,31 +67,4 @@ def update
     end
   end
 
-  def post_to_register(register_name, rsf_body)
```

Comment:
these were refactored in https://github.com/openregister/managing-registers/pull/38 but these were not removed at the time.